configure.ac

```diff
@@ -271,12 +278,21 @@ CHECK_CMA(
     [transport_cma="yes"],
     [transport_cma="no"])
 
-if test -n "$with_xpmem" -a "$with_xpmem" != "no" ; then
+# If both XPMEM and CMA requested, user needs to choose one:
+if test -n "$with_xpmem" -a "$with_xpmem" != "no" -a -n "$with_cma" -a "$with_cma" != "no" ; then
+    AC_MSG_ERROR([Cannot choose both XPMEM and CMA transports, use --with to choose one])
+# Check which was requested, XPMEM or CMA:
+elif test -n "$with_xpmem" -a "$with_xpmem" != "no" ; then
     transport_cma="no"
     AC_DEFINE([USE_XPMEM], [1], [Define if XPMEM transport is active])
 elif test -n "$with_cma" -a "$with_cma" != "no" ; then
     transport_xpmem="no"
     AC_DEFINE([USE_CMA], [1], [Define if Cross Memory Attach transport is active])
+# If neither, disable XPMEM and CMA:
+else
+    transport_xpmem="no"
+    transport_cma="no"
+    AC_MSG_RESULT([Neither XPMEM nor CMA transport requested])
```

Comment:
Same here.  Probably not necessary, but helpful imho to separate from OFI/portals.